noxfile-template.py

```diff
@@ -113,8 +181,53 @@ def py(session):
     if session.python in TESTED_VERSIONS:
         _session_tests(session)
     else:
-        print("SKIPPED: {} tests are disabled for this sample.".format(session.python))
+        session.skip("SKIPPED: {} tests are disabled for this sample.".format(
+            session.python
+        ))
+
+
+#
+# cloud_run session
+#
+
+@nox.session
+def cloud_run(session):
+    """Run tests for cloud run."""
+    if 'cloud_run' not in TEST_CONFIG['opt_in_sessions']:
+        session.skip('SKIPPED: cloud_run tests are disabled for this sample.')
+        return
+
+    if os.path.exists("requirements.txt"):
+        session.install("-r", "requirements.txt")
+
+    if os.path.exists("requirements-test.txt"):
+        session.install("-r", "requirements-test.txt")
 
+    user_envs = get_pytest_env_vars()
+
+    # Only update gcloud on Kokoro.
+    if os.environ.get("KOKORO_JOB_NAME"):
+        # Activate service account
+        key_file = os.environ['GOOGLE_APPLICATION_CREDENTIALS']
+        session.run(
+            "gcloud", "auth", "activate-service-account",
+            "--key-file={}".format(key_file)
+        )
+
+        # Set gcloud project
+        project = user_envs['GCLOUD_PROJECT']
+        session.run("gcloud", "config", "set", "project", project)
```

Comment:
I'd rather the test itself take in the environment variable and throw an error if it doesn't exist. The test can use a flag to set this so it doesn't have to permanently change the gcloud config when it's run. 

Also, I _think_ `GOOGLE_CLOUD_PROJECT` is preferred over `GCLOUD_PROJECT` IIRC

Reply:
Sure, using a command line flag seems good. I'll remove this.